### ISSUE TITLE
New-ExcelChart -LegendPosition top

### ISSUE THREAD

**Issue by jotaene** [dfinke/ImportExcel#479]
Hi

This is an extraordinary module, thanks for your work!

It seems that with -LegendPostion 'Top' I always get 'Right' position of legend.

Thanks again.

**Comment by dfinke** [comment]
Thanks! Could you please post and example of the script, xlsx, and version of the module.

**Comment by jotaene** [comment]
Yes.

ImportExcel Version: 5.3.4
xlsx: [Test.xlsx](https://github.com/dfinke/ImportExcel/files/2523169/Test.xlsx)
script: [script.txt](https://github.com/dfinke/ImportExcel/files/2523171/script.txt)

Thank you

**Comment by dfinke** [comment]
Looks like a bug. 

https://github.com/dfinke/ImportExcel/blob/master/New-ExcelChart.ps1#L404

`$LegendPosition` does not evaluate to `$true` when specified. Perhaps cause it is an Enum.

# Hacked it 

This works, but does not work when `$LegendPosition` is not specified or defaulted with "Right"

`if ($LegendPosition.ToString().Length -gt 0) { $Chart.Legend.Position    = $LegendPosition }`

Need to think about it. @jhoneill my first time hitting this with an enum, not sure that is the root cause though.

**Comment by jotaene** [comment]
Now it works. Thanks again.

**Comment by jhoneill** [comment]
Yes. It is the same bug I've introduced before 
`If ($foo) {do something} `
Fails when `$foo` is zero. I thought I'd flushed most of them out 

 I'll take a small bet that  `[OfficeOpenXml.Drawing.Chart.eLegendPosition]::top ` is zero. 
Your fix will work fine, elsewhere we've used 
`if ($PSBoundParameters.ContainsKey('LegendPosition'))`

At some point I will have to write something which gets the parameters for each function and looks to see if it contains if ($parameter)
